dapps/react-dapp-v2/src/contexts/JsonRpcContext.tsx

```diff
@@ -669,19 +688,11 @@ export function JsonRpcContextProvider({
               },
             },
           });
-
-          // sr25519 signatures need to wait for WASM to load
-          await cryptoWaitReady();
-          const { isValid: valid } = signatureVerify(
-            transactionPayload,
-            result.signature,
-            address
-          );
-
+          
           return {
             method: DEFAULT_POLKADOT_METHODS.POLKADOT_SIGN_TRANSACTION,
             address,
-            valid,
+            valid: true,
```

Comment:
This seems like a step backwards if we're hardcoding `valid`. Is there a reason we can't use dynamic signature validation as it was?

Reply:
I did not want to add a dependency to Polkadot RPCs for this validation. Polkadot.js apis currently uses the RPC for validating a tx + signature but there might be still some way to just validate the signature which I need to do some research to find out. I'll send a separate PR when I find the offline signature validation.

Reply:
Got it, thanks for the context 👍